CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/ConfigurationPane.java

```diff
@@ -157,18 +156,15 @@ public void createNewRun(final Map<String, Attribute> vertexAttributes, final Ma
     }
 
     /**
-     * Set the configuration pane to display the specified column headers and
-     * sample data rows.
+     * Set the configuration pane to display the specified column headers and sample data rows.
      *
      * @param columnLabels Column header labels.
      * @param currentData Rows of sample data.
      */
     public void setSampleData(final String[] columnLabels, final List<String[]> currentData) {
-        for (final Tab tab : tabPane.getTabs()) {
-            final RunPane runPane = (RunPane) tab.getContent();
+        tabPane.getTabs().stream().map(tab -> (RunPane) tab.getContent()).forEachOrdered(runPane -> {
             runPane.setSampleData(columnLabels, createTableRows(currentData));
-            runPane.refreshDataView();
```

Comment:
This has reverted the change from https://github.com/constellation-app/constellation/pull/783/commits/e3c9eef8de696e268acbea846aab10a8dca37008 hence now I can see the mirror image on the Delimited File Importer- Preview table.

Reply:
Good spotting!